cam1: 
  img_publisher_node:
    ros__parameters:
      image_undistored: true
      cam_id: 2
      width: 640
      height: 480
      k_new: 0.9
      K: [240.56893524485719, 0.00000000000000, 304.02167270751090,
          0.00000000000000, 240.59643968823400, 225.87784580002230,
          0.00000000000000, 0.00000000000000, 1.00000000000000]
      d: [-0.03628217932683, -0.01441553791176,  0.00795021573370, -0.00241447522342]

cam2: 
  img_publisher_node:
    ros__parameters:
      image_undistored: true
      cam_id: 6
      width: 640
      height: 480
      k_new: 0.9
      K: [242.09042455226663, 0.00000000000000, 305.27410826178294,
          0.00000000000000, 241.96309557671768, 233.84942267839520,
          0.00000000000000, 0.00000000000000, 1.00000000000000]
      d: [-0.04086438291848, -0.00722785810761, 0.00395257639926, -0.00190489762760]

cam3: 
  img_publisher_node:
    ros__parameters:
      image_undistored: true
      cam_id: 0
      width: 640
      height: 480
      k_new: 0.9
      K: [239.35312993382436, 0.00000000000000, 308.71493813687908,
          0.00000000000000, 239.59440270542146, 226.24771387864561,
          0.00000000000000, 0.00000000000000, 1.00000000000000]
      d: [-0.04211185968680, 0.00803630431552, -0.01334505838778, 0.00370625371074]

cam4: 
  img_publisher_node:
    ros__parameters:
      image_undistored: true
      cam_id: 4
      width: 640
      height: 480
      k_new: 0.9
      K: [239.35312993382436, 0.00000000000000, 308.71493813687908,
          0.00000000000000, 239.59440270542146, 226.24771387864561,
          0.00000000000000, 0.00000000000000, 1.00000000000000]
      d: [-0.04211185968680, 0.00803630431552, -0.01334505838778, 0.00370625371074]